### PR TITLE
revamp tracing/monitoring docs

### PR DESCRIPTION
**Pull request by dotansimha**

### CODE REVIEW

**Review by ardatan — APPROVED** [review]

**Review by saihaj — APPROVED** [review]
I suppose for SEO having a separate page works best 🤔 but I do see how it is much more simpler to navigate from user perspective

File: packages/web/docs/src/pages/docs/gateway/features/monitoring-tracing.mdx [hunk 1]
```diff
@@ -0,0 +1,1388 @@
...
+    graphql_envelop_request_time_summary: true,
+    graphql_envelop_phase_parse: true,
+    graphql_envelop_phase_validate: true,
+    graphql_envelop_phase_context: true,
+    graphql_envelop_error_result: true,
+    graphql_envelop_phase_execute: true,
+    graphql_envelop_phase_subscribe: true,
+    graphql_envelop_schema_change: true,
+    graphql_yoga_http_duration: true
+  }
```
Comment (EmrysMyrddin) [inline packages/web/docs/src/pages/docs/gateway/features/monitoring-tracing.mdx:791]: Metrics are now under a `metrics` field
```suggestion
  prometheus: {
    // Enable the metrics you want to expose
    // The following represent the default config of the plugin.
    metrics: {
      graphql_gateway_fetch_duration: true,
      graphql_gateway_subgraph_execute_duration: true,
      graphql_gateway_subgraph_execute_errors: true,
      graphql_envelop_deprecated_field: true,
      graphql_envelop_request: true,
      graphql_envelop_request_duration: true,
      graphql_envelop_request_time_summary: true,
      graphql_envelop_phase_parse: true,
      graphql_envelop_phase_validate: true,
      graphql_envelop_phase_context: true,
      graphql_envelop_error_result: true,
      graphql_envelop_phase_execute: true,
      graphql_envelop_phase_subscribe: true,
      graphql_envelop_schema_change: true,
      graphql_yoga_http_duration: true,
    }
  }
```

**Review by n1ru4l — APPROVED** [review]

**Review by EmrysMyrddin — APPROVED** [review]
I've pushed a commit to fix the missing `metrics` option and remove references to old option name.

Looks amazing !

**Review by n1ru4l — CHANGES_REQUESTED** [review]
please move this document out of features to the top level under gateway

**Comment by dotansimha** [comment]
@n1ru4l fixed :)

**Pull request merged**